content/en/docs/contribute/advanced.md

```diff
@@ -59,6 +57,25 @@ reviewed is usually small. These queries specifically exclude localization PRs,
   Determine whether any additional changes or updates need to be made for the PR to be merged. If you think the PR is ready to be merged, comment `/approve`.
 - [Not against master](https://github.com/kubernetes/website/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+-label%3Ado-not-merge+label%3Alanguage%2Fen+-base%3Amaster): If it's against a `dev-` branch, it's for an upcoming release. Make sure the [release meister](https://github.com/kubernetes/sig-release/tree/master/release-team) knows about it by adding a comment with `/assign @<meister's_github-username>`. If it's against an old branch, help the PR author figure out whether it's targeted against the best branch.
 
+### When to close Pull Requests
+
+Reviews and approvals are one tool to keep our PR queue short and current. Another tool is closure. 
+
+- Close any PR where the CLA hasn’t been signed for two weeks. 
+PR authors can reopen the PR after signing the CLA, so this is a low-risk way to make sure nothing gets merged without a signed CLA.
+
+- Close any PR where the author has not responded to comments or feedback in 2 or more weeks.
+
+Don't be afraid to close pull requests. Contributors can easily reopen and resume works in progress. Oftentimes a closure notice is what spurs an author to resume and finish their contribution.
+
+To close a pull request, leave a `/close` comment on the PR.
+
+{{ <note> }}
+
```

Comment:
Hi, @zacharysarah. 
nit: The `note` shortcode should get corrected:
{{< note >}}
{{< /note >}}

Reply:
Thanks, @kbhawkey!

Reply:
@kbhawkey 👋 If this looks good to you, please feel free to LGTM/approve. 🙇

Reply:
/lgtm

Reply:
@kbhawkey Remember you can `/approve` now, too. 😄